docs/source/configure.rst

```diff
@@ -10,7 +10,7 @@ When you run the command to start the Quipucords server, you supply values for s
 
 - Accepting or changing the default exposed server port
 - Selecting a directory for SSH keys
-- Selecting a directory for the SQLlite database
+- Selecting a directory for the database
```

Comment:
I think this will still only apply to SQLite.  We could indicate its not required for Postgres.